test-case/manual/check-simple-probe-capture.md

```diff
@@ -0,0 +1,86 @@
+# Probes test
+Simple probes functionality test.
+
+# Preconditions
+1. Linux kernel should have these config options enabled:
+   CONFIG_SND_SOC_SOF_DEBUG_PROBES=y
+   CONFIG_SND_SOC_SOF_HDA_PROBES=y
+
+   These options are not user selectable and can not be enabled with
+   'make menuconfig'. Edit .config manually instead.
+
+2. Firmware should have these config options enabled:
+   CONFIG_PROBE=y
+
+   The following defaults are Ok:
+   CONFIG_PROBE_POINTS_MAX=16
+   CONFIG_PROBE_DMA_MAX=4
+
+   In fact PROBE_DMA_MAX is ignored as inection is not currently supported
```

Comment:
s/inection/injection
Why mention CONFIG_PROBE_DMA_MAX in the first place?

Reply:
For me it comes with 0 as default, but never checked the kconfig file to be honest..